src/lib.rs

```diff
@@ -1,7 +1,68 @@
+#![deny(missing_docs)]
+
+//! Unofficial Rust library for the SendGrid API.
+//! 
+//! This crate requires Rust 1.15 or higher as it uses a crate that has a custom derive
+//! implementation.
+//! 
+//! sendgrid-rs implements all of the functionality of other supported SendGrid client libraries.
+//! To use sendgrid-rs you must first create a SendGrid account and generate an API key. To create
+//! an API key for your SendGrid account, use the account management interface or see the
+//! [SendGrid API Documentation](https://sendgrid.com/docs/API_Reference/Web_API_v3/API_Keys/index.html).
+//! 
+//! sendgrid-rs is available on [crates.io](https://crates.io/crates/sendgrid) and can be included
+//! in your Cargo.toml as follows:
+//!
+//! ```toml
+//! [dependencies]
+//! sendgrid = "X.X.X"
+//! ```
+//!
+//! # Features
+//! The projects has the following feature flags:
+//! * `rustls`: this feature flag switches the default SSL provider in the operating system (usually
+//! OpenSSL) with RusTLS, which is an OpenSSL reimplementation in Rust.
+//! * `native-tls`: enabled by default, this feature flag enabled the default SSL provider in the
+//! operating system (usually OpenSSL).
+//! an OpenSSL reimplementation in Rust.
```

Comment:
What do you mean here?

Reply:
Line 27 is due to bad copy pasting, I removed it :)